llvm/include/llvm/CodeGen/MachineLoopInfo.h

```diff
@@ -66,6 +67,14 @@ class MachineLoop : public LoopBase<MachineBasicBlock, MachineLoop> {
   /// it returns an unknown location.
   DebugLoc getStartLoc() const;
 
+  /// \brief Find the llvm.loop metadata for this loop.
+  /// If each branch to the header of this loop contains the same llvm.loop
+  /// metadata, then this metadata node is returned. Otherwise, if any
+  /// latch instruction does not contain the llvm.loop metadata or
+  /// multiple latch instructions contain different llvm.loop metadata nodes,
+  /// then null is returned.
```

Comment:
Why null? Should we use the max value of alignment if multiple latch instructions contain different `llvm.loop` metadata nodes.

Reply:
it is allowed to not set this loop metadata node from front end and patch wants to set a default alignment for such situation.